### PR TITLE
added a postgresql security group

### LINKED ISSUE #39: Add Postgres security group?
Hi there - Just wondering how you feel about me adding support for a Postgresql security group?  I wanted to discuss it before doing the work and submitting a PR.

### PR DESCRIPTION
**Pull request by davidpellerin**
I've added a postgresql security group very much in the same style as the MySQL security group

Closes #39

### CODE REVIEW

**Comment by antonbabenko** [comment]
Good. Few comments:
1. Use [the latest code](https://github.com/terraform-community-modules/tf_aws_sg/blob/master/sg_mysql/main.tf) as an example (split resources)
1. Use [the module README](https://github.com/terraform-community-modules/tf_aws_sg/blob/master/sg_mysql/main.tf) as an example
1. Update [main README.md](https://github.com/terraform-community-modules/tf_aws_sg/blob/master/README.md)

**Comment by davidpellerin** [comment]
@antonbabenko I've made those changes you suggested.

File: sg_postgresql/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+You can use these in your terraform template with the following steps.
+
+1. Adding a module resource to your template, e.g. `main.tf`
+
+```
+module "sg_postgresql" {
+  source = "github.com/terraform-community-modules/tf_aws_sg/sg_postgresql"
+  security_group_name = "${var.security_group_name}-postgresql"
+  vpc_id = "${var.vpc_id}"
+  source_cidr_block = "${var.source_cidr_block}"
```
Comment (antonbabenko) [inline sg_postgresql/README.md]: It should be like this - `source_cidr_block = ["${var.source_cidr_block}"]`

**Review by antonbabenko — COMMENTED** [review]
Minor comment and LGTM

**Comment by davidpellerin** [comment]
@antonbabenko good catch! thanks! I've made that change.

**Pull request merged**